app/assets/javascripts/components/profile_card_head.es6.jsx

```diff
@@ -0,0 +1,50 @@
+class ProfileCardHead extends React.Component {
+
+  static get propTypes() {
+    return {
+      cardName: React.PropTypes.string.isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      cardName: '',
+    };
+  }
+
+  get styles() {
+    return {
+      container: {
+        position: 'relative',
+        height: 40,
+        verticalAlign: 'middle',
+        width: '100%',
+      },
+      cardName: {
+        position: 'relative',
+        float: 'left',
+        lineHeight: '40px',
+        marginLeft: 5,
+        fontSize: 24,
```

Comment:
If I remember correctly was it fontSize that requires 'px' for convention? 

Why does height have no 'px' while lineHeight does? So many conventions :confounded: